src/Command/DispatchCommand.php

```diff
@@ -32,6 +32,13 @@ final class DispatchCommand extends AbstractNeedApplyCommand
     private const LABEL_NOTHING_CHANGED = 'Nothing to be changed.';
     private const FILES_DIR = 'project';
 
+    /** @var string[] */
+    private const HOOK_URLS_TO_BE_DELETED = [
+        'https://api.codacy.com',
+        'https://www.flowdock.com',
```

Comment:
What is flowdock? Never saw this on our projects

Reply:
It was only on one project and deactivated...